syntax = "proto3";

package ssn.pdfservice.v1;

option go_package = "pdfservicev1";

service PdfService {
  rpc RasterizePdf(RasterizePdfRequest) returns (stream RasterizePdfResponse) {}
}

message RasterizePdfRequest {
  bytes data = 1;
  uint32 dpi = 2;
  enum Format {
    PNG = 0;
  }
  Format format = 3;
}

message RasterizePdfResponse {
  uint32 index = 1;
  bytes data = 2;
}
